sympy/functions/special/bsplines.py

```diff
@@ -202,3 +202,60 @@ def bspline_basis_set(d, knots, x):
     """
     n_splines = len(knots) - d - 1
     return [bspline_basis(d, knots, i, x) for i in range(n_splines)]
+
+def interpolating_spline(d, x, xv='x', yv='y'):
```

Comment:
I suggest calling these X and Y as interpolating_poly does. (But with no default values, as remarked below)